tempodb/encoding/vparquet/compactor_test.go

```diff
@@ -147,8 +147,8 @@ func TestValueAlloc(t *testing.T) {
 }
 
 func TestCountSpans(t *testing.T) {
-	batchSize := rand.Intn(1000) + 1
-	spansEach := rand.Intn(1000) + 1
+	batchSize := 300 + rand.Intn(25)
```

Comment:
nit: let's leave a comment about high mem usage when these values are set to big numbers.

Reply:
same for `tempodb/encoding/vparquet2/compactor_test.go`